doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
> The signal is ignored while the error is not

@Trott wait a minute, what do you mean by ignoring the signal?

also what do you think about my idea from just a few minutes ago in the issue discussion? https://github.com/nodejs/node/issues/2853#issuecomment-140275010